setuptools/command/easy_install.py

```diff
@@ -319,7 +319,7 @@ def finalize_options(self):
                     self.all_site_dirs.append(normalize_path(d))
         if not self.editable:
             self.check_site_dir()
-        self.index_url = self.index_url or "https://pypi.python.org/simple"
+        self.index_url = self.index_url or "https://pypi.org/simple"
```

Comment:
@ewdurbin @brainwane CCed for sanity check - is this the right thing to do?

Also, does this mean that the old versions of `setuptools` out there now are going to start breaking at the end of the week?

Reply:
> Also, does this mean that the old versions of setuptools out there now are going to start breaking at the end of the week?

I don't think so. The linked blog states:

> The address pypi.python.org will continue to redirect to Warehouse.

Reply:
I might suggest changing the default to `https://pypi.org/simple/` to save a redirect.

```shell
$ curl -I https://pypi.org/simple
HTTP/2 301 
location: https://pypi.org/simple/
```

But otherwise this is 👍.

The `pypi.python.org` service will continue to operate for the foreseeable future as it is currently:

```shell
$ curl -I https://pypi.python.org/simple
HTTP/2 301 
location: https://pypi.org/simple
```

```shell
$ curl -I https://pypi.python.org/simple/
HTTP/2 301 
location: https://pypi.org/simple/
```